### ISSUE TITLE
How do I import TFVC repos from an old TFS, into Git repos?

### ISSUE THREAD

**Issue by Rod-at-DOH** [MicrosoftDocs/azure-devops-docs#13189]
I need to migrate several TFVC repos into new Git repos in GitHub. Our TFS is version 2015, and is on-prem, behind our firewall. I've researched this topic. I came across this article [Import from TFVC](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/repos/git/import-from-TFVC.md) but reading it a couple times it appears to me to describe a situation where the TFVC repo is in Azure DevOps and the target Git repo is also in Azure DevOps. That doesn't describe our situation. Is there a Microsoft Doc that addresses our situation?

**Comment by WilliamAntonRohm** [comment]
@Rod-at-DOH -- Rod, thank you for your feedback and question.  Please consider these resources, if you haven't already:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

This article may help:
- [Migrate from TFVC to Git](https://learn.microsoft.com/en-us/devops/develop/git/migrate-from-tfvc-to-git)

cc @steved0x for awareness

I am closing this issue now.  You are welcome to @ mention me for any followup.  We hope to hear from you again.

**Comment by Rod-at-DOH** [comment]
Hi @WilliamAntonRohm I do have some questions/concerns. The "Import repository from TFVC to Git" link is one I've looked at before but thought it wouldn't apply to our situation. There's a note at that link that says:

> The feature documented in this article requires TFS 2017 Update 2 or later version.

We've got TFS 2015, so are we simply out of luck?

The second concern I have is posting anything related to TFS on Stack Overflow. Every time I've done that there's one guy who will instantly, vote my questions down, then deny all replies/answers to my questions. He's made my life, as the TFS administrator, hell. I'll try asking, but I strongly suspect this guy will treat me with the same disdain he has in the past. Which means I'll not get my questions answered nor any help. Thus, why I ask here.
